### PR TITLE
docs: update broken links & latest/stable

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

follow-up for #16951 

<details>
  <summary><b>Before submitting</b></summary>

- Was this **discussed/agreed** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- Did you make sure to **update the documentation** with your changes? (if necessary)
- Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- Did you list all the **breaking changes** introduced by this pull request?
- Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

</details>

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

<details>
  <summary>Reviewer checklist</summary>

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

</details>

cc @borda @carmocca @akihironitta @tchaton @justusschock @awaelchli

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/2_refactor.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ body:
 
         - [**Metrics**](https://github.com/Lightning-AI/metrics):
           Machine learning metrics for distributed, scalable PyTorch applications.
-        - [**Lite**](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.html):
+        - [**Lite**](https://lightning.ai/docs/pytorch/latest/starter/lightning_lite.html):
```
Comment (akihironitta) [inline .github/ISSUE_TEMPLATE/2_refactor.yaml:36]: I get `404`, but is it going to be available after some other PR lands?

**Review by akihironitta — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/2_refactor.yaml [hunk 2]
```diff
@@ -33,7 +33,7 @@ body:
...
           enables pure PyTorch users to scale their existing code on any kind of device while retaining full control over their own loops and optimization logic.
         - [**Flash**](https://github.com/Lightning-AI/lightning-flash):
```
Comment (carmocca) [inline .github/ISSUE_TEMPLATE/2_refactor.yaml:38]: Lets remove the unmaintained projects?

File: .github/ISSUE_TEMPLATE/2_refactor.yaml (hunk 1 above)
Comment (Borda) [reply .github/ISSUE_TEMPLATE/2_refactor.yaml:36]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/2_refactor.yaml (hunk 2 above)
Comment (Borda) [reply .github/ISSUE_TEMPLATE/2_refactor.yaml:38]:
```suggestion
```

Comment (Borda) [reply .github/ISSUE_TEMPLATE/2_refactor.yaml:38]: lets do it in separate PR

File: .github/ISSUE_TEMPLATE/2_refactor.yaml (hunk 1 above)
Comment (carmocca) [reply .github/ISSUE_TEMPLATE/2_refactor.yaml]: There are more uses of this link in this PR

File: .github/ISSUE_TEMPLATE/3_feature_request.yaml [hunk 3]
```diff
@@ -39,7 +39,7 @@ body:
 
         - [**Metrics**](https://github.com/Lightning-AI/metrics):
           Machine learning metrics for distributed, scalable PyTorch applications.
-        - [**Lite**](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.html):
+        - [**Lite**](https://lightning.ai/docs/pytorch/latest/starter/lightning_lite.html):
```
Comment (carmocca) [inline .github/ISSUE_TEMPLATE/3_feature_request.yaml:42]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/4_documentation.yaml [hunk 4]
```diff
@@ -22,7 +22,7 @@ body:
 
         - [**Metrics**](https://github.com/Lightning-AI/metrics):
           Machine learning metrics for distributed, scalable PyTorch applications.
-        - [**Lite**](https://pytorch-lightning.readthedocs.io/en/latest/starter/lightning_lite.html):
+        - [**Lite**](https://lightning.ai/docs/pytorch/latest/starter/lightning_lite.html):
```
Comment (carmocca) [inline .github/ISSUE_TEMPLATE/4_documentation.yaml:25]:
```suggestion
```

**Review by carmocca — APPROVED** [review]

**Review by lantiga — APPROVED** [review]

**Pull request merged**